system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -40,16 +40,27 @@
       opacity: 0;
       transition: opacity 0.2s $photon-easing;
 
-      &:focus, &:active {
+      &[aria-expanded='true'] {
+        background-color: $info-active-color;
+        border-radius: 1px; // The shadow below makes this the desired larger radius
+        box-shadow: 0 0 0 5px $info-active-color;
+        fill: $fill-primary;
+      }
+
+      &[aria-expanded='true'] + .info-option {
```

Comment:
With the refactoring above, this should be able to be nested in the above section `&[aria-expanded='true'] {` as `+ .info-option {`

Reply:
Resolved with commit https://github.com/mozilla/activity-stream/pull/3751/commits/ff5b0da5e9f764c46a7c62e5780c891cb1d268f3!